test/envoye2e/driver/fake_stackdriver.go

```diff
@@ -191,9 +204,98 @@ func (s *LoggingServer) GetLogEntries(w http.ResponseWriter, req *http.Request)
 	}
 }
 
+// ListTraces implements ListTraces method.
+func (s *TracesServer) ListTraces(context.Context, *cloudtracev1.ListTracesRequest) (*cloudtracev1.ListTracesResponse, error) {
+	s.mux.Lock()
+	defer s.mux.Unlock()
+	for traceID, trace := range s.traceMap {
+		s.listTracesResp.Traces = append(s.listTracesResp.Traces, trace)
+		delete(s.traceMap, traceID)
```

Comment:
No need to delete it I think. This is just for testing, no need to prevent leak.